### PR TITLE
option to split by percentage

### PR DESCRIPTION
**Pull request by Nisheanthan**
-Added radio button to choice in between percentage and general split
-Displaying appropriate note to user for error situations.

### CODE REVIEW

File: costSplit/index.html [hunk 1]
```diff
@@ -1,36 +1,103 @@
...
+      const name2 = document.getElementById('name2');
+      const percentage1 = document.getElementById('percentage1');
+      const percentage2 = document.getElementById('percentage2');
+      const button = document.querySelector('button');
+      const textField = document.querySelector("#textField");
+
+      radioBtn.addEventListener('click',showPercentageDiv);
+      button.addEventListener('click', splitAmount);
+
+      function showPercentageDiv() {
```
Comment (Neha) [inline costSplit/index.html]: You should use ES6 arrow function

File: costSplit/index.html [hunk 2]
```diff
@@ -1,36 +1,103 @@
...
+        } else {
+          radioBtn.value = 'false';
+          radioBtn.checked = false;
+          percentageSplit.style.display = 'none';
+          generalSplit.style.display = 'block';
+        }
+        textField.innerHTML = '';
+      }
+
+      function splitAmount() {
```
Comment (Neha) [inline costSplit/index.html]: You should use ES6 arrow function

File: costSplit/index.html [hunk 3]
```diff
@@ -1,36 +1,103 @@
...
+      <p id="textField"></p>
+    </section>
+
+    <script>
+      const radioBtn = document.getElementById('radioBtn');
+      const generalSplit = document.getElementById('general-split');
+      const percentageSplit = document.getElementById('percentage-split');
+      const noOfPeople = document.getElementById("people");
+      const splitCostField = document.getElementById("splitCost");
+      const name1 = document.getElementById('name1');
```
Comment (Neha) [inline costSplit/index.html]: We should avoid names like name1, name2, percentage1, percentage2

File: costSplit/index.html [hunk 4]
```diff
@@ -1,36 +1,103 @@
...
-  
-  button.addEventListener('click', () => {
-    let amount = Number(amountField.value);
-    let splitCost = Number(splitCostField.value);
-    
-    const indvAmt = calculateAmount(amount, splitCost);
+  <body>
+    <section class="container">
+      <h2>Cost Spliter</h2>
+      <label>Split By Percentage: </label>
```
Comment (Neha) [inline costSplit/index.html]: This should be before submit button. As of now the UI is confusing.

File: costSplit/index.html [hunk 5]
```diff
@@ -1,36 +1,103 @@
...
+          generalSplit.style.display = 'block';
+        }
+        textField.innerHTML = '';
+      }
+
+      function splitAmount() {
+        if (splitCostField.value) {
+          let splitCost = Number(splitCostField.value);
+          if (radioBtn.checked) {
+            if (percentage1.value && percentage2.value) {
```
Comment (Neha) [inline costSplit/index.html]: Error handling to check the percentage value is adding to 100 is missing.

**Review by Neha — CHANGES_REQUESTED** [review]
Thank you for taking time to raising the PR.

I have added few code changes. Please review them.

File: costSplit/index.html (hunk 5 above)
Comment (Nisheanthan) [reply costSplit/index.html]: I think it is handled in line 73, Am I wrong?

**Pull request closed without merging**